ocaml/idl/datamodel.ml

```diff
@@ -1494,7 +1496,10 @@ let _ =
   error Api_errors.usb_group_contains_pusb ["pusbs"]
     ~doc:"The USB group contains active PUSBs and cannot be deleted." ();
   error Api_errors.usb_group_contains_no_pusbs ["usb_group"]
-    ~doc:"The USB group does not contain any PUSBs." ()
+    ~doc:"The USB group does not contain any PUSBs." ();
+  error Api_errors.too_many_vusbs [ "vusbs" ]
+    ~doc:"You reached the maximal number of vusbs." ()
```

Comment:
I believe I've said it before - maybe the change is lost? This should be `VUSBs`.

Reply:
updated.